editor/project.clj

```diff
@@ -220,7 +217,7 @@
                                                     [org.clojure/test.check   "0.9.0"]
                                                     [org.clojure/tools.trace  "0.7.9"]
                                                     [org.mockito/mockito-core "1.10.19"]
-                                                    [ring "1.4.0"]]
+                                                    [ring "1.9.6"]]
```

Comment:
Somehow it was also necessary to also update `ring` (used in the updater test). Shared jetty dependencies...